DHT.h

```diff
@@ -37,11 +37,28 @@ written by Adafruit Industries
 
 class DHT {
   public:
-   DHT(uint8_t pin, uint8_t type, uint8_t count=6);
+   // Note that count is now ignored as the DHT reading algorithm adjusts itself
+   // based on the speed of the processor.
+   DHT(uint8_t pin, uint8_t type, uint8_t count) {
+     DHT(pin, type);
```

Comment:
Thank you for your feedback.

I am not sure I understand your comment.

The 3 arg constructor DHT(pin, type, count) calls the 2 arg constructor with DHT(pin, type) in the header.  The 2 arg constructor is the header definition for the 2 arg constructor in the DHT.cpp file.  I removed the 3 arg constructor that used to be in the DHT.cpp file.

Does that make sense or am misunderstanding your comment?